### PR TITLE
9385: Add Admin AC tests

### PR DESCRIPTION
**Pull request by PhysicsJon**
[AB#9385](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/9385)

### CODE REVIEW

File: src/buyingcatalogue/Chart.yaml [hunk 1]
```diff
@@ -29,11 +29,11 @@ dependencies:
   version: ~0.1.0
 - name: bapi-db-deploy
   condition: bapi-db-deploy.enabled
-  version: 1.24.9
+  version: 1.24.12
```
Comment (gavinwakeley1) [inline src/buyingcatalogue/Chart.yaml:32]: We wouldn't usually update the core component versions outside of a release

File: src/buyingcatalogue/Chart.yaml [hunk 2]
```diff
@@ -29,11 +29,11 @@ dependencies:
...
   repository: "@gpitfuturesdevacr"
 - name: bapi
   condition: bapi.enabled
-  version: 1.24.9
+  version: 1.24.12
```
Comment (gavinwakeley1) [inline src/buyingcatalogue/Chart.yaml:36]: We wouldn't usually update the core component versions outside of a release

File: src/buyingcatalogue/Chart.yaml [hunk 3]
```diff
@@ -69,7 +69,7 @@ dependencies:
   repository: "@gpitfuturesdevacr"
 - name: mp
   condition: mp.enabled
-  version: 1.26.3
+  version: 1.26.5
```
Comment (gavinwakeley1) [inline src/buyingcatalogue/Chart.yaml:72]: We wouldn't usually update the core component versions outside of a release

**Review by gavinwakeley1 — CHANGES_REQUESTED** [review]
Looks good - just want to confirm the updates to the Chart.yaml

File: src/buyingcatalogue/Chart.yaml (hunk 1 above)
Comment (PhysicsJon) [reply src/buyingcatalogue/Chart.yaml:32]: I'll revert this change

File: src/buyingcatalogue/Chart.yaml (hunk 2 above)
Comment (PhysicsJon) [reply src/buyingcatalogue/Chart.yaml:36]: I'll revert this change

File: src/buyingcatalogue/Chart.yaml (hunk 3 above)
Comment (PhysicsJon) [reply src/buyingcatalogue/Chart.yaml:72]: I'll revert this change

**Review by gavinwakeley1 — DISMISSED** [review]
Looks good Jon

**Review by gavinwakeley1 — APPROVED** [review]
Looks good Jon

**Pull request merged**